examples/fuzzing/rt_n12/fuzz.py

```diff
@@ -0,0 +1,173 @@
+from qiling import *
+from qiling.const import *
+from qiling.extensions import afl
+from typing import Optional
+from qiling.os.const import *
+from qiling.extensions.coverage import utils as cov_utils
+import argparse
+
+payload_location = 0 # location in snapshot to be modified
+
+def my_send(ql:Qiling):
+    """
+    Hook to avoid writing to now no-longer connected socket
+    used to create this snapshot
+    """
+    params = ql.os.resolve_fcall_params({'sockfd': INT,'buf':POINTER, 'len':SIZE_T, 'flags':INT })
+    ql.log.info(buf)
+    return
```

Comment:
No need to use an empty `return` statement at the end of the function.

Reply:
Removed